docs/source/pages/DM3390.rst

```diff
@@ -36,26 +36,26 @@ In addition, IMU (over SPI) sensor is also added to this OAK FFC board.
 Key features
 ************
 
-- 6x 2-line MIPI camera interfaces (all of the CCMs supported in DepthAI can run on 2-lane MIPI, most of them also in full resolution)
-- :ref:`OAK-SoM-Max` connected to the baseboard
-- 5V power input via barrel jack
-- USB 3.1 Gen 1 Type-C
-- IMU support (BMI270 by default, footprint for BNO086)
-- uSD card slot
-- Onboard EEPROM
-- Pads for DepthAI SoM 1.8V SPI
-- Pads for DepthAI SoM 1.8V Aux Signals (I2C, UART, GPIO)
-- PSRBS connector (5V, STROBE, RST, BOOT_SEL, FSIN_ALL, FSIN_STEREO)
-- 1025 coin battery holder
-- Design files produced with Altium Designer 22
+- 6x 2-line MIPI camera interfaces (all of the CCMs supported in DepthAI can run on 2-lane MIPI, most of them also in full resolution).
+- :ref:`OAK-SoM-Max` connected to the baseboard.
+- 5V power input via barrel jack.
+- USB 3.1 Gen 1 Type-C.
+- IMU support (BMI270 by default, footprint for BNO086).
+- uSD card slot.
+- Onboard EEPROM.
+- Pads for DepthAI SoM 1.8V SPI.
+- Pads for DepthAI SoM 1.8V Aux Signals (I2C, UART, GPIO).
+- PSRBS connector (5V, STROBE, RST, BOOT_SEL, FSIN_ALL, FSIN_STEREO).
+- 1025 coin battery holder.
+- Design files produced with Altium Designer 22.
```

Comment:
LGTM, except this part. On all product pages we would have **Key features** listed without dots, except here?

Reply:
My mistake there. I'd like to use periods when we have lists of complete sentences, but agree not necessary in feature lists like this.